ui/src/components/view/UploadResourceIcon.vue

```diff
@@ -195,7 +195,7 @@ export default {
       this.options.img = ''
     },
     beforeUpload (file) {
-      if (!/\.(svg|jpg|jpeg|png|bmp|SVG|JPG|PNG)$/.test(file.name)) {
+      if (!/\.(svg|jpg|jpeg|png|bmp|SVG|JPG|JPEG|PNG)$/.test(file.name)) {
```

Comment:
Should we also check for `BMP` ?

Reply:
@jbampton yes adding BMP will also be good

Reply:
well,
```suggestion
      if (!/\.(svg|jpg|jpeg|png|bmp|SVG|JPG|JPEG|PNG|BMP)$/.test(file.name)) {
```
then...?

Reply:
@DaanHoogland  The changes are good

Reply:
Better to have a case insensitive regex to match all variations of the file extensions.

Even a `.JPeg` is valid it seems

Reply:
@jbampton 
```suggestion
      if (!/\.(SVG|JPG|JPEG|PNG|BMP)$/.test(file.name.tuUpperCase())) {
```
?